GenumKit/Stencil/Filters.swift

```diff
@@ -80,12 +80,32 @@ struct StringFilters {
 
 struct ArrayFilters {
   static func join(value: Any?) throws -> Any? {
-    guard let array = value as? [Any] else { throw FilterError.InvalidInputType }
-    let strings = array.flatMap { $0 as? String }
-    guard array.count == strings.count else { throw FilterError.InvalidInputType }
+    let strings: [String]
+    if let array = value as? [Any] {
+      strings = array.flatMap { $0 as? String }
+      guard array.count == strings.count else { throw FilterError.InvalidInputType }
+    } else if let array = value as? [String] {
+      strings = array
+    } else {
+      throw FilterError.InvalidInputType
+    }
 
     return strings.joinWithSeparator(", ")
   }
+
+  static func joinAnd(value: Any?) throws -> Any? {
```

Comment:
It has been a while since I checked how Stencil evolved and what new features were brought by more recent versions, but I remember that @kylef wanted to implement filters with parameters. In case that already landed we could use that instead of defining a dedicated filter for each joiner string we need!